### PR TITLE
release(v0.2.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by Unix4ever**
This is the official v0.2.0-alpha.0 release.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

**Review by smira — COMMENTED** [review]
to make it really cool, we can do Makefile/.drone.yml changes to push changelog to the release

**Review by smira — COMMENTED** [review]
also metadata.yml should be updated: https://github.com/talos-systems/cluster-api-bootstrap-provider-talos/pull/90

**Comment by smira** [comment]
> to make it really cool, we can do Makefile/.drone.yml changes to push changelog to the release

more details: https://github.com/talos-systems/cluster-api-bootstrap-provider-talos/commit/db60f9eb0697c4949be9c00cf8dc7787d383bad2

**Comment by Unix4ever** [comment]
> also metadata.yml should be updated: talos-systems/cluster-api-bootstrap-provider-talos#90

I guess that should be done in follow up update

**Comment by rsmitty** [comment]
nah, it has to be done before cutting the alpha release or clusterctl won't work.

**Comment by Unix4ever** [comment]
yeah, now that makes sense :)

**Review by smira — COMMENTED** [review]
I don't see `Makefile` changes to generate release notes

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: docker.io/unix4ever/cluster-api-control-plane-talos-controller
```
Comment (smira) [inline config/manager/kustomization.yaml:7]: I don't think this should be there?

**Review by rsmitty — APPROVED** [review]

**Review by smira — APPROVED** [review]

**Pull request merged**